pkg/scheduler/framework/plugins/podtopologyspread/filtering.go

```
@@ -194,10 +194,9 @@ func (pl *PodTopologySpread) RemovePod(ctx context.Context, cycleState *framewor
 func getPreFilterState(cycleState *framework.CycleState) (*preFilterState, error) {
 	c, err := cycleState.Read(preFilterStateKey)
 	if err != nil {
-		// The preFilterState wasn't pre-computed in prefilter. We ignore the error for now since
-		// we are able to handle that by computing it again (e.g. in Filter()).
+		// The preFilterState wasn't pre-computed in prefilter.
 		klog.V(5).Infof("Error reading %q from cycleState: %v", preFilterStateKey, err)
-		return nil, nil
+		return nil, err
```

Ditto. We can simply return a verbose err and remove the logging.